pkg/aws/ec2/ec2.go

```diff
@@ -142,6 +159,61 @@ func (c *Client) describeNetworkInterfaces(ctx context.Context, subnets ipamType
 	return result, nil
 }
 
+// describeNetworkInterfacesFromInstances lists all ENIs matching filtered EC2 instances
+func (c *Client) describeNetworkInterfacesFromInstances(ctx context.Context) ([]ec2_types.NetworkInterface, error) {
+	ENIsFromInstances := make(map[string]*struct{})
+
+	instanceAttrs := &ec2.DescribeInstancesInput{}
+	if len(c.instancesFilters) > 0 {
+		instanceAttrs.Filters = c.instancesFilters
+	}
+
+	paginator := ec2.NewDescribeInstancesPaginator(c.ec2Client, instanceAttrs)
+	for paginator.HasMorePages() {
+		c.limiter.Limit(ctx, "DescribeNetworkInterfacesFromInstances")
```

Comment:
The name here refers to the AWS API:
```suggestion
		c.limiter.Limit(ctx, "DescribeInstances")
```

Reply:
updated